README.md

```diff
@@ -24,7 +24,8 @@ module.exports = {
 
     new WasmPackPlugin({
       crateDirectory: path.resolve(__dirname, "crate"),
-      withTypeScript: true
+      withTypeScript: true, // false by default
+      noInstallMode: false, // false by default, it will avoid wasm-bindgen installation. Use it only if you know what you're doing!
```

Comment:
Maybe we can add a more generic option, something like: `extraArgs: "--mode no-install"` that will pass the option to the cli.

Reply:
For sure this will make the plugin more maintainable as we wouldn't have to care about wasm-pack api changes but the user would have to be somehow directed to the wasm-pack docs. Maybe by just inserting a  comment with the link in the example code will be enough.

I'll wait for your confirmation in order to change it. 😊

Reply:
Sorry I don't understand your proposition?

I agree that it's more maintainable for us but less user friendly.

Reply:
Sorry, my explanation was very poor. What I meant is that in order to allow the user learning about the options that he has available we should point him to the [wasm-pack docs](https://rustwasm.github.io/wasm-pack/book/commands/build.html) by using the `readme` or even a comment in code (like the ones that we have above in line 27 & 28). I guess that would be enough although reviewing the  [wasm-pack docs](https://rustwasm.github.io/wasm-pack/book/commands/build.html) I haven't found any reference to the `--no-typescript` flag.

The change I was referring to was just changing the code according to your initial suggestion.

Reply:
Ah yes I absolutely agree.

Reply:
A mention in the README sounds good to me.